docs/_templates/autosummary/class.rst

```diff
@@ -0,0 +1,33 @@
+{#
+   We show all the class's methods and attributes on the same page. By default, we document
+   all methods, including those defined by parent classes.
+-#}
+
+{{ objname | escape | underline }}
+
+.. currentmodule:: {{ module }}
+
+.. autoclass:: {{ objname }}
+   :no-members:
+   :no-inherited-members:
+   :no-special-members:
```

Comment:
We have to set this to avoid the methods being duplicated. That's because we configured `.. autoclass` in `conf.py` to now show inherited members